app/Models/PluginVersion.php

```diff
@@ -143,13 +154,31 @@ protected function renderBlade(Entity $entity)
 
         $html = str_replace([
             '@php', '@dd', '@inject', '@yield', '@section', '@auth', '@guest', '@env', '@once', '@push', '@csrf',
-            '@include', '\Illuminate\\',
+            '@include', '\Illuminate\\', "@i18n('"
         ], [
-            '', '', '', '', '', '', '', '', '', '', ''
+            '', '', '', '', '', '', '', '', '', '', '', '', '', "trans('*."
         ], $html);
         $html = preg_replace('`dd\((.*?)\)`i', '', $html);
         $html = preg_replace('`config\((.*?)\)`i', '', $html);
+        $locale = auth()->user()->locale;
+        $lines = [];
+
+        //Replace translation keys of existing translations
+        foreach ($this->getTranslationsAttribute() as $translation) {
```

Comment:
You can replace these whole two blocks with a single `preg_replace_content` like it's done for `@liveAttribute` and 3 lines :-)

Reply:
`preg_replace_callback` doesnt seem to be working, do I leave the current implementation while its fixed?